packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -317,9 +317,9 @@ class AndroidCameraCameraX extends CameraPlatform {
   @override
   Future<void> dispose(int cameraId) async {
     preview?.releaseFlutterSurfaceTexture();
-    liveCameraState?.removeObservers();
+    unawaited(liveCameraState?.removeObservers());
     processCameraProvider?.unbindAll();
-    imageAnalysis?.clearAnalyzer();
+    unawaited(imageAnalysis?.clearAnalyzer());
```

Comment:
@bparrishMines this makes me wonder if I did some wrapping wrong. Should all of the methods I wrapped that were `void` been wrapped as `Future<void>`?